File: Python/tests/test_ts_akam.py
import pytest

from hbayesdm.models import ts_akam


def test_ts_akam():
    _ = ts_akam(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ts_akam.R
context("Test ts_akam")
library(hBayesDM)

test_that("Test ts_akam", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ts_akam(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
